CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ Released on June 2, 2020.
 
 ### Task Library
 
-- None
+- Fix PostgresExecute task auto commit when commit is set to `False` - [#2658](https://github.com/PrefectHQ/prefect/issue/2658)
```

Comment:
The way we update the changelog has changed. Use the process outlined here https://github.com/PrefectHQ/prefect/tree/master/changes